### ISSUE TITLE
Proposal for adjusting various constraints in the CRT pallet

### ISSUE THREAD

**Issue by dobertRowneySr** [Joystream/joystream#5098]
#### TLDR
Add proposal proposal in order to update project token pallet storage constant used as constraints (for example the minimum amm curve slope and max patronage rate)

#### Support
- Many constraining values needed for security configuration have hardcoded values set at the development stage and not throughly discussed (partly because we did not have apriori evidence for the quality of this values)
- We want to enable the governance to explicitly set values for these constraints, as we will implicitly assume that the usage of this feature will provide some a posteriori guidelines for adjusting these constraints

#### Details
- `min_amm_slope = 1_000_000u128` lower bound on the `slope` amm parameter to prevent supply dilution
- `max_patronage_rate = 15%` (yearly rate) upper mound on the patronage yearly interest rate to prevent dilution
- `amm_buy_fee = amm_sell_fees = 3 perthousand`
-  `sale_platform_fee = 2 %`
-  `min_revenue_split_duration = 5 blocks`
-  `min_revenue_time_to_start = 0 block` (# blocks in between revenue share issuance block and actual revenue share starting block)
- `amm_deactivation_threshould = 0` upper bound on % of the CRT supply that must be met by the AMM minted CRT supply in order for the creator token to deactivate the AMM

Also work on this has already been started in #4485
